tests/security/secureboot/kernel_lockdown.pm

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Verify that if we are "secure booted" that kernel lockdown is enabled
+#
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#109611
+
+use strict;
+use warnings;
+use base 'opensusebasetest';
+use testapi;
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Make sure system is secureboot enabled
+    validate_script_output('mokutil --sb-state', sub { m/SecureBoot enabled/ });
+    assert_script_run('cat /sys/kernel/security/lockdown');
+    my $lockdown_info = script_output('grep "\[none\]" /sys/kernel/security/lockdown', proceed_on_failure => 1);
```

Comment:
Why use `proceed_on_failure` here instead of just `assert_script_run`?

Or even better, just use `validate_script_output`.

Reply:
hmm, I can see there are 3 options in /sys/kernel/security/lockdown file:
none [integrity] confidentiality

So I use a simple way, the expected result should not be "[none]"
let me enhance it.